serverless/src/scaleImage/utils/cmr/fetchCmrConcept.js

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios'
+import { getEarthdataConfig } from '../../../../../sharedUtils/config'
+import { determineEarthdataEnvironment } from '../../../util/determineEarthdataEnvironment'
+import { getSystemToken } from '../../../util/urs/getSystemToken'
+
+/**
+ * Retrieve the CMR metadata for the provided concept id
+ * @param {String} conceptId A collection or granule concept-id
+ * @returns {JSON} The collection metadata associated with the provided concept id
+ */
+export const fetchCmrConcept = async (conceptId) => {
+  const headers = {}
+  const earthdataEnvironment = determineEarthdataEnvironment()
+
+  if (!process.env.IS_OFFLINE) {
```

Comment:
Here we are getting the system token from the `db` that function also will pull down a new system token if the one in there is older than 1 day. We do this in case the collection is behind an ACL in which case we would need a token to grab the necessary metadata

Reply:
Would there ever be a time where a user is requesting an image that they don't have access to? Presumably this is coming from a result from within EDSC which means it has come back to the user, using their token or its open. I hate system tokens, and it seems like we could stop using it here because now we have access to the users access token, which we didn't have when this was a separate application.

Reply:
Yea I considered this so no there isn't really a way for a user to do that because the collections or granules that come back from CMR are already going to be bounded by the ACLS so then we are just forwarding the concept Id to the browse-scaler lambda. The issue with the token is that the current way that this is setup the GET request is coming off an `<img>` component we are just passing the `url` to that component through this request transformation. So we'd have to either somehow pass a header in the GET request that the img is making when its looking for the `src` value or otherwise; pass the TOKEN as a url parameter which we would not want to do